build.gradle

```diff
@@ -11,6 +11,10 @@ buildscript {
 description = 'RxJava: Reactive Extensions for the JVM – a library for composing asynchronous and event-based programs using observable sequences for the Java VM.'
 
 apply plugin: 'java'
+	compileJava {
+		sourceCompatibility = '1.8'
```

Comment:
Ah, okay, thanks for the consideration!

On Tue, Jun 28, 2016 at 10:43:43AM -0700, Artem Zinnatullin wrote:
> In build.gradle:
> 
> > @@ -11,6 +11,10 @@ buildscript {
> >  description = 'RxJava: Reactive Extensions for the JVM – a library for composing asynchronous and event-based programs using observable sequences for the Java VM.'
> >
> >  apply plugin: 'java'
> > +     compileJava {
> > +             sourceCompatibility = '1.8'
> 
> RxJava can't target Java 8, it's Java 6+ library, mostly because of Android.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

-- 
Raffi Khatchadourian, PhD
Assistant Professor and Open Educational Resources (OER) Fellow
Computer Systems Technology
New York City College of Technology
City University of New York
300 Jay St., Room N913
Brooklyn, NY 11201-1909
United States
+1 (718) 260-5325
rkhatchadourian@citytech.cuny.edu
http://cuny.is/khatchad